looper/__main__.py

```diff
@@ -1,10 +1,16 @@
 import sys
 
-from .looper import main
+from .looper import main as looper_main
+from .divvy import main as divvy_main
 
 if __name__ == "__main__":
     try:
-        sys.exit(main())
+        if len(sys.argv) > 1 and sys.argv[1] == "divvy":
```

Comment:
Should the `console_script` be able to specify the function so that this logic is not needed?

you have them specifying different functions there, so how can they both end up here and need this logic?

Reply:
yes, you're right, removed